packages/cerebral/src/utils.js

```diff
@@ -142,4 +147,70 @@ export function getProviders (module) {
   )
 }
 
+export function dependencyMatch (changes, dependencyMap, currentKey = '') {
+  let currentMatches = []
+
+  for (const key in changes) {
+    const pathKey = currentKey ? currentKey + '.' + key : key
+
+    let matches = []
+    if (changes[key] === true) {
+      if (dependencyMap[pathKey]) {
+        matches = matches.concat(dependencyMap[pathKey])
+      }
+      if (dependencyMap[pathKey + '.*']) {
+        matches = matches.concat(dependencyMap[pathKey + '.*'])
+      }
+      if (dependencyMap[pathKey + '.**']) {
```

Comment:
Needed internally and does not affect load... just fine to use it. Think we can handle this simply by communicating it :)